modules/installation/pages/uyuni-server-setup.adoc

```diff
@@ -0,0 +1,171 @@
+[[server-setup]]
+= Uyuni Server Setup
+
+This section covers {productname} Server setup, using these procedures:
+
+* Start {productname} setup with {yast}
+* Create the main administration account with the {productname} {webui}
+* Name your base organization and add login credentials
+* Synchronize the {sle} product channel from {scc}
+
+
+
+== Set up {productname} with {yast}
```

Comment:
This is incomplete. It's not explained to the user how to get the packages installed first :-)

For stable: https://www.uyuni-project.org/pages/stable-version.html
For master: https://www.uyuni-project.org/pages/devel-version.html

It's basically the same procedure with different repositories, but I am not sure you you want to handle when to show stable and when master. Maybe it's something to be discussed with @jcayouette

Reply:
For this, we now have install-uyuni.adoc.